include/zephyr/drivers/bluetooth/hci_driver.h

```diff
@@ -21,7 +21,10 @@
 #include <zephyr/net/buf.h>
 #include <zephyr/bluetooth/buf.h>
 #include <zephyr/bluetooth/hci_vs.h>
-#include <zephyr/device.h>
+
+
+/* Opaque type representing a Zephyr device. */
```

Comment:
I think it's clearer to just say where it's defined, instead of explaining what it is.

```suggestion
/* From <zephyr/device.h>: */
```